### PR TITLE
[#IOPAE-327] Add check on wildcard for enable all users to manage key

### PR DESCRIPTION
**Pull request by giuseppedipinto**
We have a feature _(manage key)_ that we don't enable for users EXCEPT the ones in an enabling list _(MANAGE_FLOW_ENABLE_USER_LIST config key)_.
The enabling list is a list of strings that represents a user id.

The new feature is that **_"If one of such strings is "*", we suddenly open the feature to everyone."_**.

**Note:**
_A change on `io-infra > core > devportal.tf` is required in order to allow all users_.
_Change `MANAGE_FLOW_ENABLE_USER_LIST` value to `*`_.

#### List of Changes

- subscriptions controller

#### Motivation and Context

We need to rollout MANAGE API KEY to all APIM users.
#### How Has This Been Tested?

Unit Tests and integration test with local environment (be/fe).
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by balanza — CHANGES_REQUESTED** [review]
> TL;DR please document the wildcard

We should be cautious on this kind of changes. When reading this PR I saw it's just a one-line modification, so I assumed it was just an improvement.
Digging deeper, I found we were adding a whole new, undocumented behavior to the application. 

These kind of modifications are potentially dangerous not for themselves (the feature is actually well-tested), but for future devs which might not be aware of it. **All the knowledge relies on an arbitrary, single-character string** (`*`) buried into a js-specific mechanism (to check the index of an element into an array of string).

Without questioning the solution, I think we should at least comment `manageFlowEnableUserList` variable on `config.ts`, explaining we accept a wildcard character and how it affects the behavior of the application (readers should understand without dive deeper into code).

-----

Let me spend a couple of lines about the design of the solution. I'm not proposing an alternative, I just aim to stimulate some thoughts.

What we are coding is:
```
We have a feature (manage key) that we don't enable to user EXCEPT the ones in an enabling list.
The enabling list is a list of strings which represents a user id.
If one of such strings is "*", we suddenly open to feature to everyone.
```
Which might lead to configurations like:
```
MANAGE_FLOW_ENABLE_USER_LIST = ["01GC77MF2WYY52DCSQXVEDCE74", "01GJMF341BZQBP71Q39S1EHBH6", "6kYC77MF2WYY56TVSQXVEDCE90", "04JJMF341BZQBP71Q39S1EHF38", "01GC77MF2WYY52DCSQXVEDCE74", "01GJMF341BZQBP71Q39S1EHBH6", "6kYC77MF2WYY56TVSQXVEDCE90", "04JJMF341BZQBP71Q39S1EHF38", "01GC77MF2WYY52DCSQXVEDCE74", "01GJMF341BZQBP71Q39S1EHBH6", "6kYC77MF2WYY56TVSQXVEDCE90", "04JJMF341BZQBP71Q39S1EHF38","01GC77MF2WYY52DCSQXVEDCE74", "01GJMF341BZQBP71Q39S1EHBH6", "*", "6kYC77MF2WYY56TVSQXVEDCE90", "04JJMF341BZQBP71Q39S1EHF38", "01GC77MF2WYY52DCSQXVEDCE74", "01GJMF341BZQBP71Q39S1EHBH6", "6kYC77MF2WYY56TVSQXVEDCE90", "04JJMF341BZQBP71Q39S1EHF38"]
```
Question: you are a developer debugging this application; would you understand this feature is open to anyone?

**Review by balanza — APPROVED** [review]
